lib/Access.php

```diff
@@ -651,6 +651,8 @@ private function ldap2ownCloudNames($ldapObjects, $isUsers) {
 						? $ldapObject[$sndAttribute][0] : '';
 					$this->cacheUserDisplayName($ocName, $nameByLDAP, $sndName);
 				}
+			} else {
+				Util::writeLog('user_ldap', "Setting the owncloud name for LDAP entry {$ldapObject['dn'][0]} isn't possible", Util::ERROR);
```

Comment:
logging now done when the OutOfBounds excaption in https://github.com/owncloud/user_ldap/blob/master/lib/User/UserEntry.php#L106-L110 is finally logged. Also see https://github.com/owncloud/core/pull/28805 to allow skipping of "broken" users during sync